_guides/managing-oauth-clients.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Managing OAuth Clients
+layout: article
+section: Developing Components
+order: 1
+since: 20190403
+---
+This document reveals the new approach to OAuth utilization within the platform
+and explains how to [manage](#managing-oauth-client) OAuth clients for
+components in a tenant.
+
+## Changes
+
+
+OAuth client was defined for a component in its environment variables. This way
+global access to the component could not be enabled, because users in other tenants would see the
+component and its keys. Now OAuth clients are defined
+separately from the components’ environment variables, allowing the users in the
+other tenant to see all available components without security breach.
+
+To manage OAuth clients the user requires the following permissions:
+
+
+`tenants.oauth_clients.get`
+
+`tenants.oauth_clients.edit`
+
+`tenants.oauth_clients.create`
+
+`tenants.oauth_clients.delete`
+
+
+To acquire these permissions, please contact support.
+
+## Managing OAuth Client
+
+To initiate work with OAuth client, first specify `"useOAuthClient": true` in the _component.json_ file. For compatibility reasons, we should also create a link between the new implementation and environment variables if they were present.
```

Comment:
I'm not sure that "To initiate work with OAuth client" tells everybody what exactly you want to say. Please give an introduction that we are going to talk about components that are using OAuth clients.

Also following is not really clear. What is the new implementation? Please be more concrete. Mention how it was before first, then how it is done here.

> .... link between the new implementation and environment variables ...